### DISCUSSION TITLE
Transfer Process Fails with "Policy not found for contract"

### DISCUSSION THREAD

**Discussion by obermeier** [eclipse-tractusx/eclipse-tractusx.github.io#578] [Q&A]
Hi,
thank you very much for this nice tutorials.

While experimenting with the tutorial content, I encountered an "issue" during the transferprocess step. [See](https://github.com/eclipse-tractusx/eclipse-tractusx.github.io/blob/83c2e28f8628358e240549110e9e058b2542bc37/docs/tutorials/e2e/boost/consumeData.md?plain=1#L103C12-L103C12)

The transfer (.../v2/transferprocesses) fails with 
``` "errorDetail": "Policy not found for contract: 3",```

It works for me with the "contractAgreementId" from  {{CONSUMER_MANAGEMENT_URL}}/v2/contractnegotiations/request

Have I done something wrong, or is this a small bug in the tutorial?

BR,
Stefan

**Comment by KMA-2019** [comment]
@obermeier Thanks for your Feedback. 
@jkbquabeck Can you look into this within section consumeData and create issue if needed - thanks!

**Reply by stephanbcbauer** [reply]
@jkbquabeck "It works for me with the "contractAgreementId" from {{CONSUMER_MANAGEMENT_URL}}/v2/contractnegotiations/request" => do we have to document this in another way in our tutorial?

@obermeier Do you receive data with your fix?

**Reply by stephanbcbauer** [reply]
@DaEwe putting you in the loop

**Comment by obermeier** [comment]
> @jkbquabeck "It works for me with the "contractAgreementId" from {{CONSUMER_MANAGEMENT_URL}}/v2/contractnegotiations/request" => do we have to document this in another way in our tutorial?
> 
> @obermeier Do you receive data with your fix?

Yes, something like this [1] as the response of the request and the access informations on the endpoint defined in ```receiverHttpEndpoint```

```
{
    "@type": "IdResponse",
    "@id": "4e671e30-bf6f-4629-9bd1-9cf235f0312a",
    "createdAt": 1705600004455,
    "@context": {
        "@vocab": "https://w3id.org/edc/v0.0.1/ns/",
        "edc": "https://w3id.org/edc/v0.0.1/ns/",
        "tx": "https://w3id.org/tractusx/v0.0.1/ns/",
        "odrl": "http://www.w3.org/ns/odrl/2/"
    }
}
```

**Reply by stephanbcbauer** [reply]
To be honest, for me, it's unclear whether there is still a problem or not. If your issue remains, please let me know where you need assistance. 

Meanwhile, I will check if we have to adapt the documentation. Any suggestions are welcome.

I will also test the whole tutorial again and see if there is a issue.

**Reply by stephanbcbauer** [reply]
@obermeier , @tmberthold, @jkbquabeck what is the current status here? What about the references issue? Thank you very much

**Reply by obermeier** [reply]
IMHO clould be closed. Since it appeared only in my setup.
